src/modules/mc_pos_control/MulticopterPositionControl.cpp

```diff
@@ -190,6 +190,28 @@ void MulticopterPositionControl::parameters_update(bool force)
 					    "Manual speed has been constrained by maximum speed", _param_mpc_xy_vel_max.get());
 		}
 
+		if (_param_mpc_vel_man_back.get() > _param_mpc_vel_manual.get()) {
+			_param_mpc_vel_man_back.set(_param_mpc_vel_manual.get());
```

Comment:
```suggestion
			_param_mpc_vel_man_back.set(-1.f);
```
Why not reset it, if it's aynways out of range? Just an idea, maybe there's a disadvantage.

Reply:
I think it's more intuitive for the user if the value gets clamped to the maximum instead of setting it back to -1.